docker-compose.custom.yml

```
@@ -46,6 +46,9 @@ services:
       # https://github.com/developmentseed/eoAPI/issues/16
       # - TITILER_ENDPOINT=raster
       - TITILER_ENDPOINT=http://127.0.0.1:8082
+      # PgSTAC extensions
+      #- EOAPI_STAC_EXTENSIONS='["filter", "query", "sort", "fields", "pagination", "context", "transaction"]'
```

```suggestion
      #- EOAPI_STAC_EXTENSIONS=["filter", "query", "sort", "fields", "pagination", "context", "transaction"]
```

I think we need to remove the single quotes, otherwise there is a JSON parsing error occurring

---

both are supported

---

also if you pass this as environment variable you will have to use the quote if I remember well

---

Have no idea why, I just noticed that as is, it fails to deploy in my machine, with 

```
docker compose -f docker-compose.custom.yml --profile gunicorn up
```

giving 

```
pydantic.env_settings.SettingsError: error parsing env var "eoapi_stac_extensions"
```

If I remove the single quotes like suggested, it works though 🤷‍♂️

---

well it doesn't 😅 this must be a docker compose thing 👍